torch/nn/utils/rnn.py

```diff
@@ -405,5 +406,5 @@ def pack_sequence(sequences, enforce_sorted=True):
     Returns:
         a :class:`PackedSequence` object
     """
-    lengths = [v.size(0) for v in sequences]
+    lengths = torch.as_tensor([v.size(0) for v in sequences])
```

Comment:
Are there existing issues filed for the underlying issues causing these modifications to be necessary?

Reply:
Yes, the `pack_padded_sequence` only accepts Tensor in TorchScript, see https://github.com/pytorch/pytorch/blob/a4716d0e26485d36679e140c00105d3775bd76eb/torch/nn/utils/rnn.py#L196, so it won't accept an list[int] here